src/javascript/binary/common_functions/form_validation.js

```diff
@@ -234,11 +236,21 @@ const Validation = (() => {
         if (field.$error && field.$error.length) {
             field.$error.setVisibility(0);
         }
+
+        if (field.$error_address && field.$error.length) {
+            field.$error_address.setVisibility(0);
+        }
     };
 
     const showError = (field, message) => {
-        clearError(field);
-        field.$error.text(localize(message)).setVisibility(1);
+        if (field.selector === '#address_line_1' || field.selector === '#address_line_2') {
+            clearError(field);
+            field.$error.text(localize(message)).setVisibility(1);
+            field.$error_address.setVisibility(1);
+        } else {
+            clearError(field);
+            field.$error.text(localize(message)).setVisibility(1);
```

Comment:
line 251 & 252 are repeated in both if and else (line 247 and 248). we can move them out and call it regardless of any condition.